src/main/java/org/terasology/snakeTournament/systems/SnakeTournamentAuthoritySystem.java

```diff
@@ -20,7 +20,6 @@
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
-import org.terasology.logic.health.DoDamageEvent;
```

Comment:
Probably still need this in here.  Just need to add a dependency on Core in the module.txt.